sentry_zendesk/__init__.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.conf import settings  # noqa
+import logging
+
+
+try:
+    VERSION = __import__('pkg_resources') \
```

Comment:
😢 

Importing `pkg_resources` is quite expensive (like, really). Consider using `setuptools_scm` for automatic versioning instead, see an example [here](https://github.com/ESSS/conda-devenv/pull/39).